client/collabor8/src/app/projects/page.tsx

```diff
@@ -39,77 +39,41 @@ export default function MyProjects() {
           <div className="projects-page__filters">
             <ProfileCard {...user} />
           </div>
+
           <div className="projects-page__projects">
             {ownerProjects.length > 0 && (
```

Comment:
HTML is broken on this page, it doesn't show headers correctly for:

1. "Owner Projects"
2. "Team member in"
3. "Finished tasks in"
4. Should show: "Not participating in any projects" if user has no projects

Reply:
I fixed it and pushed it again. :)